roles/restore/tasks/secrets.yml

```diff
@@ -54,6 +54,37 @@
       no_log: "{{ no_log }}"
   when: secrets['postgresConfigurationSecret']['data']['type'] | b64decode == 'managed'
 
+- name: Set new receptor secret names
+  set_fact:
+    previous_receptor_ca_name: "{{ previous_deployment_name }}-receptor-ca"
+    previous_receptor_tls_name: "{{ previous_deployment_name }}-receptor-work-signing"
+  no_log: "{{ no_log }}"
+
+- name: Set new name for receptor secrets using deployment_name
```

Comment:
The purpose of this block is to edit the json dictionary of secrets so that the deployment name matches that of the new deployment (`deployment_name`), not that of the old deployment (`previous_deployment_name`)